firmware/gerrit_watcher/gerrit_watcher.ino

```diff
@@ -13,10 +15,13 @@ const auto PASSWORD = "your_password";
 const String GERRIT_URL = "http://your_gerrit_url:8080";
 const auto GERRIT_USERNAME = "your_gerrit_username";
 const auto GERRIT_HTTP_PASSWORD = "your_gerrit_http_password";
+const auto MY_EFFECT = Effect::PULSE;
```

Comment:
This should be outside the `if` statement as it's nothing that needs to be in a non-version controlled header file. :smile: